src/logger/index.ts

```diff
@@ -58,28 +56,34 @@ export default class ConsoleLogger implements ILogger {
   };
 
   public succeedIcon = (message: string): void => {
-    this.info(`${logSymbols.success} ${message}`);
+    this.info(`✅ ${message}`);
   };
 
   public failIcon = (message: string): void => {
-    this.error(`${logSymbols.error} ${message}`);
+    this.error(`❌ ${message}`);
   };
 
   public warnIcon = (message: string): void => {
-    this.warn(`${logSymbols.warning} ${message}`);
+    this.warn(`⚠️ ${message}`);
   };
 
   public infoIcon = (message: string): void => {
-    this.info(`${logSymbols.info} ${message}`);
+    this.info(`ℹ️ ${message}`);
   };
 
   public spinner = (message: string): ISpinner => {
+    const that = this;
     if (this.oraInstance) {
       this.oraInstance.stop();
     }
-    this.oraInstance = ora(message).start();
-    this.spinnerActive = true;
-    this.oraInstance.start();
+    import('ora')
```

Comment:
This dynamic import is a workaround and not a good one.  It causes race conditions that are more challenging to solve than to convert to using a different spinner.  Need to evaluate alternatives.